lib/core/versions/latest/models/ProtocolParameters.ts

```diff
@@ -18,8 +18,8 @@ export default interface ProtocolParameters {
   maxNumberOfTransactionsPerTransactionTime: number;
   /** Maximum operations per batch. */
   maxOperationsPerBatch: number;
-  /** Maximum size of an operation in bytes. */
-  maxOperationByteSize: number;
+  /** Maximum size of the `operationData` property in bytes. */
+  maxOperationDataByteSize: number;
```

Comment:
Suggestion: can we put the unit at the end? `maxOperationDataSizeInBytes`

Reply:
Sure!